### ISSUE TITLE
[1.21.3] Missing textures in model neoforge:bucket_milk#inventory:

### ISSUE THREAD

**Issue by Lolothepro** [neoforged/NeoForge#1641]
**Minecraft Version:** 1.21.3

**NeoForge Version:** 21.3.4-beta

**Logs:** https://mclo.gs/m9nrSbZ

**Steps to Reproduce:**
1. Load the game
2. Check the log file

**Description of issue:**

```
[Worker-Main-5/WARN] [minecraft/ModelManager]: Missing textures in model neoforge:bucket_milk#inventory:
    minecraft:textures/atlas/blocks.png:neoforge:items/bucket_base
    minecraft:textures/atlas/blocks.png:neoforge:items/bucket_cover
    minecraft:textures/atlas/blocks.png:neoforge:items/bucket_fluid
```
NeoForge causes this warning

**Comment by sciwhiz12** [comment]
The area around those model files for the milk buckets and dynamic buckets (as I understand it) needs quite a bit of attention for cleanup. 

The textures originally refered from `neoforge:bucket_milk` are different from the mask textures now present in the assets folder; see https://github.com/neoforged/NeoForge/tree/4ea73399de818f3670a78a5afd5772a05a73221d/src/main/resources/assets/forge/textures/items for the original textures that the model referred to.

Some of these models also need some investigation: why do we duplicate `item/generated` and `item/handheld` as `neoforge:item/default` and `neoforge:item/default-tool`? Are these models even used at the moment?

The models ought to be moved to datageneration, with comments on why they exist as well for future reference.

**Comment by Lolothepro** [comment]
fixed in 1.21.4
